installer/templates/phx_web/components/core_components.ex

```diff
@@ -237,6 +197,7 @@ defmodule <%= @web_namespace %>.CoreComponents do
     assigns
     |> assign(field: nil, id: assigns.id || field.id)
     |> assign(:errors, Enum.map(errors, &translate_error(&1)))
+    |> assign_new(:id, fn -> field.id end)
```

Comment:
Is this not a duplicate of what’s happening two lines above?

Reply:
it is

Reply:
I may be missing something, but why is it there? Surely it will never be called because of the assign two lines above?

Reply:
I changed the code and simply forgot to remove it.

Reply:
Sorry, was in a meeting previously, so my reply was a bit too short. The initial response should have been more like:

"it is, I just forgot to remove it. Will push a fix soon" 😃

Reply:
Thanks. I was just reading through the code. Your response made me question my understanding of assign_new ;-)